.github/workflows/Shared-Deploy-On-RiseClipseWeb.yml

```diff
@@ -0,0 +1,79 @@
+# *************************************************************************
+# **  Copyright (c) 2022 CentraleSupélec & EDF.
+# **  All rights reserved. This program and the accompanying materials
+# **  are made available under the terms of the Eclipse Public License v2.0
+# **  which accompanies this distribution, and is available at
+# **  https://www.eclipse.org/legal/epl-v20.html
+# ** 
+# **  This file is part of the RiseClipse tool
+# **  
+# **  Contributors:
+# **      Computer Science Department, CentraleSupélec
+# **      EDF R&D
+# **  Contacts:
+# **      dominique.marcadet@centralesupelec.fr
+# **      aurelie.dehouck-neveu@edf.fr
+# **  Web site:
+# **      https://riseclipse.github.io
+# *************************************************************************
+
+name: Deploy on RiseClipseWeb
+
+# This CD workflow push the newly released jar to RiseClipseWeb
+name: Deploy JAR
+
+on:
+  workflow_call:
+    inputs:
+    dest_folder:
+      description: Destination folder on the remote machine
+      required: true
+
+env:
+  REMOTE_HOST: ${{ secrets.REMOTE_HOST }}
+  REMOTE_USER: ${{ secrets.REMOTE_USER }}
+  REMOTE_KEY: ${{ secrets.REMOTE_KEY }}
```

Comment:
Why is the key needed?

Reply:
For ssh connection to the remote in the scp step ([appleboy/scp-action](https://github.com/appleboy/scp-action)). Can use key_path too

Reply:
Why use an SSH key instead of a password?

Reply:
Yes we could use a password, doing this.